beta-src/src/state/game/game-api-slice.ts

```diff
@@ -178,17 +196,10 @@ const startNewOrder = (
     };
     setCommand(state, command, "unitCommands", prevUnitID);
   }
-  state.order.inProgress = true;
-  state.order.unitID = unitID;
-  state.order.orderID =
-    orderID ||
-    gameData.currentOrders.find((order) => order.unitID === unitID)?.id;
-  state.order.onTerritory = onTerritory;
-  state.order.toTerritory = toTerritory;
   delete state.order.type;
-  if (type) {
-    state.order.type = type;
-  }
+  const orderData = getDataForOrder(state, action.payload);
+  state.order = orderData;
```

Comment:
Can we just inline this above

`state.order = getDataForOrder(state, action.payload);`

Reply:
No, I need orderData in its own variable because I need to get unitID from it below.